### ISSUE TITLE
SQL Time Data type translated

### ISSUE THREAD

**Issue by ngzero** [MicrosoftDocs/feedback#1297]
**Link to article:**
https://docs.microsoft.com/de-de/sql/t-sql/data-types/time-transact-sql?view=sql-server-2017

**Problem:**
The navigation item to the left shows "Uhrzeit" but it should be "time" because it's the data type.

**Comment by MikeRayMSFT** [comment]
@ngzero - thank you for submitting. Reviewing the issue with our localization team.

**Comment by SonjaSaltzman** [comment]
@nokura @DavidPrio found this by accident...

**Comment by CeciAc** [comment]
Hello, @ngzero
 
Thank you for your feedback.
 
Our linguistic team reviewed your linguist suggestions and accepted them. We implemented your feedback and published to the updated articles on the Docs site to the following pages https://docs.microsoft.com/de-de/sql/t-sql/data-types/time-transact-sql?view=sql-server-2017
We also made a full revision of the page and fixed other linguistic issues as well.

We truly appreciate your contribution to our articles and we encourage you to continue providing your valuable feedback.
	
Kind regards,
Microsoft DOCS International Team
